wandb/integration/keras/keras.py

```diff
@@ -48,22 +53,43 @@ def is_generator_like(data):
 
 
 def patch_tf_keras():
+
     from tensorflow.python.eager import context
-    from tensorflow.python.keras.engine import training
-
-    try:
-        from tensorflow.python.keras.engine import training_arrays
-        from tensorflow.python.keras.engine import training_generator
-    except ImportError:
-        from tensorflow.python.keras.engine import training_arrays_v1 as training_arrays
-        from tensorflow.python.keras.engine import (
-            training_generator_v1 as training_generator,
-        )
+
+    from tensorflow import __version__ as tf_version
+
+    if parse_version(tf_version) >= parse_version("2.6.0"):
+        keras_engine = "keras.engine"
+
+        from keras.engine import training
+        from keras.engine import training_arrays_v1 as training_arrays
+        from keras.engine import training_generator_v1 as training_generator
+
+    else:
+        keras_engine = "tensorflow.python.keras.engine"
+
+        from tensorflow.python.keras.engine import training
+
+        try:
+            from tensorflow.python.keras.engine import (
+                training_arrays_v1 as training_arrays,
+            )
+            from tensorflow.python.keras.engine import (
+                training_generator_v1 as training_generator,
+            )
+        except (ModuleNotFoundError, AttributeError):
```

Comment:
I'm curious why we switched from ImportError to ModuleNotFoundError

Reply:
oh... my bad... i'll change it to ImportError